File: result/translate/transcoder/python2java/java/FIND_SUM_ODD_FACTORS_NUMBER.java
public static int f_gold ( int n ) {
  int res = 1 ;
  while ( n % 2 == 0 ) n = n / 2 ;
  for ( int i = 3 ;
  i < ( int ) Math . sqrt ( n ) + 1 ;
  i ++ ) {
    int count = 0 ;
    int currSum = 1 ;
    int currTerm = 1 ;
    while ( n % i == 0 ) {
      count ++ ;
      n = n / i ;
      currTerm *= i ;
      currSum += currTerm ;
    }
    res *= currSum ;
  }
  if ( n >= 2 ) res *= ( 1 + n ) ;
  return res ;
}

File: result/translate/chatgpt/java2python/python/FIND_SUM_ODD_FACTORS_NUMBER.py
def f_gold(n):
    res = 1
    while n % 2 == 0:
        n = n // 2
    i = 3
    while i <= int(n ** 0.5):
        count = 0
        curr_sum = 1
        curr_term = 1
        while n % i == 0:
            count += 1
            n = n // i
            curr_term *= i
            curr_sum += curr_term
        res *= curr_sum
        i += 1
    if n >= 2:
        res *= (1 + n)
    return res